### PR TITLE
Bugfix/1536/fix styles on page and profile

### PR DESCRIPTION
**Pull request by Skydry333**
## Description
I fixed styles in the pages: Home, 404 and Profile
1. Сentered the arrow on the items and made it bigger
#### Screenshots
|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** до ** | ** після ** |

2. changed the link on the third tab of the slider
#### Screenshots
|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** до ** | ** після ** |

3. fixed burger menu
#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** до ** | ** після ** |
4. changed the background of product cards that did not correspond to the mokap of the dark theme
#### Screenshots
|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** до ** | ** після ** |
5. the fourth switch incorrectly displayed information in Ukrainian
#### Screenshots
|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** до ** | ** після ** |
6. reduced the distance between the sentence "we are in social networks" and icons of social networks
#### Screenshots
|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** до ** | ** після ** |
7. fixed some little bugs on page 404
8. fixed fonts, added font-weight to basic headers, and removed border-radius from password and mail confirmation fields
#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** до ** | ** після ** |

### Checklist

- [ ] 🔽 My branch is up-to-date with "development" branch
- [ ] ✅All tests passed locally
- [ ] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [ ] 🔗 Link pull request to issue

### CODE REVIEW

File: src/components/model-item/model-item.style.js [hunk 1]
```diff
@@ -85,9 +86,10 @@ export const useStyles = makeStyles((theme) => ({
     fontSize: '14px',
     fontWeight: '600',
     display: 'flex',
+    alignItems: 'center',
     '& svg': {
       marginLeft: '5px',
-      fontSize: '1.2em'
+      fontSize: '1.7em'
```
Comment (boris-giga) [inline src/components/model-item/model-item.style.js:92]: please use similar dimensions
check your other files for this issue

File: src/locales/en/common.json [hunk 2]
```diff
@@ -78,7 +78,8 @@
     "horondiStyle": "Horondi style",
     "constructor": "Create by yourself",
     "catalog": "Catalog",
-    "slider": "Slider"
+    "slider": "Slider",
+    "models": "Our modals"
```
Comment (boris-giga) [inline src/locales/en/common.json:82]: did you mean "Our models"?

File: src/locales/en/home.json [hunk 3]
```diff
@@ -7,6 +7,7 @@
   "models": "Our Models",
   "allModels": "All models",
   "readMore": "Read more",
+  "aboutUs": "About us",
```
Comment (boris-giga) [inline src/locales/en/home.json:10]: check for this field in already created files, don't create duplications. 
same for other translations.

File: src/locales/ua/not-found-page.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "linkBack": "На попередню сторінку",
   "message": "Ой! Щось пішло не так",
-  "toHome": "Сторінку, яку ти шукав, не знайдено",
+  "toHome": "Сторінку, яку ти шукав, не знайдено.",
   "linkHome": "На головну"  
 }
```
Comment (boris-giga) [inline src/locales/ua/not-found-page.json:6]: please, add a new line

**Review by boris-giga — CHANGES_REQUESTED** [review]

File: src/components/model-item/model-item.style.js (hunk 1 above)
Comment (Skydry333) [reply src/components/model-item/model-item.style.js]: i got it

File: src/locales/en/common.json (hunk 2 above)
Comment (Skydry333) [reply src/locales/en/common.json]: yes i did)

File: src/locales/en/home.json (hunk 3 above)
Comment (Skydry333) [reply src/locales/en/home.json]: ok, i got it

File: src/locales/ua/not-found-page.json (hunk 4 above)
Comment (Skydry333) [reply src/locales/ua/not-found-page.json]: ok

**Review by boris-giga — APPROVED** [review]

**Review by ElitKondor — APPROVED** [review]

**Pull request merged**